web/src/components/studies/StudyDetails.tsx

```diff
@@ -0,0 +1,265 @@
+    }
+  };
+
+  const handleFeedbackSubmit = () => {
+    handleUpdateStudyStatus("Rejected");
+    toggleShowFeedbackForm();
+  };
+
+  const handleFeedbackChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setFeedback(event.target.value);
+  };
+
+  useEffect(() => {
+    const calculateRiskScore = () => {
+      // todo: how to determine if they have data?
+      let score = 0;
+
+      if (study.involves_data_processing_outside_eea) score += 10;
+      if (study.requires_dbs) score += 5;
+      if (study.requires_dspt) score += 5;
+      if (study.involves_third_party && !study.involves_mnca) score += 5;
+      if (study.involves_nhs_england || study.involves_cag) score += 5;
+
+      setRiskScore(score);
+    };
+
+    calculateRiskScore();
+    setApprovalStatus(study.approval_status);
+  }, [study]);
+  return (
+    <>
+      <Box>
+        <div className={styles["pre-description"]}>
+          <span>
+            Last updated: <span className={styles["grey-value"]}>{formatDate(study.updated_at)}</span>
+          </span>
+          <span>
+            Risk Score: <span className={styles["risk-score"]}>{riskScore}</span>
+          </span>
+          <StudyStatusBadge status={approvalStatus} isAdmin={true} />
+        </div>
+        <h2>{study.description}</h2>
+        <div>
+          <dl className={styles.ownership}>
+            <dd>
+              Owner: <span className={styles["grey-value"]}>{study.owner_username}</span>
+            </dd>
+            <dd>
+              Admins:
+              {study.additional_study_admin_usernames.length > 0 && (
+                <span className={styles["grey-value"]}>{study.additional_study_admin_usernames}</span>
+              )}
+            </dd>
+            <dd>
+              Data Controller:{" "}
+              <span className={styles["grey-value"]}>{study.data_controller_organisation.toUpperCase()}</span>
+            </dd>
+          </dl>
+
+          {study.feedback && (
+            <Alert type={"warning"} className={styles["feedback-alert"]}>
+              <AlertMessage>
+                <h4>This study has been rejected and the following feedback has been provided:</h4>
+                <p>{study.feedback}</p>
+              </AlertMessage>
+              <hr></hr>
+              <small>
+                <em>Please adjust as appropriate and request another review.</em>
+              </small>
+            </Alert>
+          )}
+
+          <h3>Additional Information</h3>
+          <dl className={styles.grouping}>
+            <h4>Sponsorships & Approvals</h4>
+            {study.involves_ucl_sponsorship && (
+              <dd className={`${styles.badge} ${styles["badge-no-risk-associated"]}`}>UCL Sponsorship</dd>
+            )}
+            {study.involves_cag && (
+              <dd className={`${styles.badge} ${styles["badge-risk-associated"]}`}>
+                CAG approval <InfoTooltip text="increases risk score by 5" />
```

Comment:
probably here and below it's worth templating values into the string, so they're defined once?